### ISSUE TITLE
Overlapping subset and class names: EMSL, Sample ID

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/sheets_and_friends#16]

**Comment by mslarae13** [comment]
@turbomam  can you add more context here? I'm curious what this is referring to

**Comment by turbomam** [comment]
> @turbomam can you add more context here? I'm curious what this is referring to

Yeah! You, @sujaypatil96 and I should be careful that we don't create multiple elements in the NMDC DataHarmonizer schema that share the same name. I think the most likely cases now are [DH sections](https://docs.google.com/spreadsheets/d/1cMlPKgjZh-v21aMYCm9x1TxzE5BwGQptBQcvuaYAtC8/edit#gid=1977494462), which are implemented as sub-**slots** of `dh_section` [DH interfaces](https://docs.google.com/spreadsheets/d/1cMlPKgjZh-v21aMYCm9x1TxzE5BwGQptBQcvuaYAtC8/edit#gid=1963017587) (sub-**classes** of `dh_interface`) and the [mixins](https://docs.google.com/spreadsheets/d/1cMlPKgjZh-v21aMYCm9x1TxzE5BwGQptBQcvuaYAtC8/edit#gid=1963017587) for  things like EMSL and JGI terms.

There are some LinkML sanity checks that can detect and report name conflicts in some cases, but sometimes it's too late when we detect them, in the sense that it's confusing to assigning new, non-conflicting names

Also, @cmungall isn't in love with my use of the term "DH interface" for a web page with columns for some combination of an environmental package and various user facility columns. Damion uses the term "template" for that sort of thing, but he also uses the term "template" for other things, too.
